### ISSUE TITLE
Jako uživatel chci zakládat tématické bloky, abych mohl své eventy třídit a zobrazovat si relevantní eventy

### ISSUE THREAD

**Issue by xverl00** [jirkae/hobby_hub#45]

**Comment by MartinBuna** [comment]
User Story #45
**Název:** Jako uživatel chci zakládat tématické bloky, abych mohl své eventy třídit a zobrazovat si relevantní eventy
**Popis:** Uživatel bude mít možnost filtrovat jednotlivé události podle tématických bloků

**Comment by JanHavel** [comment]
Tem blok = kategorie ?

**Comment by xverl00** [comment]
ano, to jsou kategorie. hashtagy jsou nepřesný protože tam není žádná centralizace a my stěží budeme dělat algoritmus hledání podobnosti ;)

**Comment by fixiee** [comment]
zmena US - chci mit moznost pridat hash tagy do udalosti, tridit je zatim nechci !!!!!!!
